pages/agent/v3/securing.md.erb

```diff
@@ -45,6 +45,10 @@ Command line evaluation can be disabled by setting [`no-command-eval`](/docs/age
 
 As plugins execute in the same way as local hooks, they can pose a potential security risk. If you're using third party plugins, you'll be executing the third party's code on your agent.
 
+You can disable plugins with the command line flag: `--no-plugins`.  
```

Comment:
I think we  should keep both of the "disable plugins" options together. Let's link the "command  line flag one also"

![image](https://user-images.githubusercontent.com/95874/130590423-6a6af1da-eb92-4245-a702-0f4f29e2dbad.png)

Note: we might need to talk about some docs on how flags and settings can be used

Reply:
Moved stuff around, but don't understand the part about linking. Let's discuss it today.